Source/WebCore/platform/graphics/cocoa/GraphicsContextGLCocoa.mm

```diff
@@ -113,13 +113,7 @@ static bool platformSupportsMetal()
     auto device = adoptNS(MTLCreateSystemDefaultDevice());
 
     if (device) {
-#if PLATFORM(MAC) || PLATFORM(MACCATALYST)
-        // Old Macs, such as MacBookPro11,4 cannot use WebGL via Metal.
-        // This check can be removed once they are no longer supported.
-        return [device supportsFamily:MTLGPUFamilyMac2];
```

Comment:
Strictly speaking we don't support builds on non Mac2 hw, but this would be still good to preserve for documentation. Also, the mac codebase doesn't make sense otherwise.  So instead, Could you preserve this, and have logic: "no metal" -> "failure". My example amends have an example